go/ct/driver/probe.go

```diff
@@ -193,7 +193,7 @@ func testState(specification spc.Specification, state *st.State, evm ct.Evm) err
 	defer result.Release()
 
 	if !expected.Eq(result) {
-		return fmt.Errorf(formatDiffForUser(state, result, expected, rules[0].Name))
+		return fmt.Errorf("%v", formatDiffForUser(state, result, expected, rules[0].Name))
```

Comment:
```suggestion
		return fmt.Error(formatDiffForUser(state, result, expected, rules[0].Name))
```
I think the linter does not suggested adding `%v` for default formatting but rather calling the error function which does not support formatting. 
Also for the other occurrences.

Reply:
well, the linter is saying that if we call `Errorf` then we have to use `Errorf("%someverb", somevalue)` and if we call `Error` then we use `Error(somevalue)`. both ways fix the problem. But yes, in this case I think we can also just call `Error`

Reply:
would it work with %w ?  for other errors with format?

Reply:
`fmt.Error` does not existe, instead using `errors.New`